example/src/App.tsx

```diff
@@ -9,27 +9,62 @@ import {
   SendStx,
 } from './components';
 import { useLocalStorage } from './hooks';
+import { useCallback, useState } from 'react';
+import GetBtcBalance from './components/GetBtcBalance';
+import GetRunesBalance from './components/GetRunesBalance';
+import { ConnectButtonsContainer } from './App.styles';
 
 function App() {
   const [network, setNetwork] = useLocalStorage<BitcoinNetworkType>(
     'network',
     BitcoinNetworkType.Mainnet
   );
-  const [addressInfo, setAddressInfo] = useLocalStorage<Address[]>('addresses', []);
+  const [addressInfo, setAddressInfo] = useState<Address[]>([]);
```

Comment:
storing the data in localStorage is necessary to persist the data across refreshes

Reply:
Thanks, forgot to change it back, had to clear it a few times while debugging